python/datasources/census_pop_estimates_sc.py

```diff
@@ -0,0 +1,195 @@
+from ingestion.standardized_columns import Race
+from datasources.data_source import DataSource
+from ingestion import gcs_to_bq_util
+import ingestion.standardized_columns as s
```

Comment:
we import this as `std_col` everywhere else, can we keep that pattern for now

Reply:
sure. I got tired of typing lol